### DISCUSSION TITLE
Welcome to saifu-issues Discussions!

### DISCUSSION THREAD

**Discussion by dvcrn** [saifuwallet/saifu#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cryptcyber** [comment]
Hello, I just found out saifu wallet and noticed that there had been no development since 2022. Are you still developing this? Thank you!

**Reply by dvcrn** [reply]
Hey, Saifu development is currently on pause indefinitely
